### PR TITLE
add badges to the README

### PR DESCRIPTION
**Pull request by ipa-rwu**
#48 add badges to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Yak
 
+[![GitHub Actions status](https://github.com/ros-industrial/yak/workflows/CI/badge.svg?branch=devel)](http://build.ros.org/job/Kdev__universal_robot__ubuntu_xenial_amd64)
```
Comment (gavanderhoorn) [inline README.md]: This says "Github actions", but then links to the ROS buildfarm.

Is this correct?

Comment (ipa-rwu) [reply README.md]: Sorry, I updated

**Review by ipa-nhg — APPROVED** [review]
Thanks :+1: 
LGTM https://github.com/ipa-rwu/yak/blob/48-update-readme/README.md

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Yak
 
+[![GitHub Actions status](https://github.com/ros-industrial/yak/workflows/CI/badge.svg?branch=devel)](https://github.com/ros-industrial/yak/actions)
+[![GitHub issues open](https://img.shields.io/github/issues/ros-industrial/yak.svg?)](https://github.com/ros-industrial/yak/issues)
+[![license - MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![support level: community](https://img.shields.io/badge/support%20level-community-lightgray.png)](http://rosindustrial.org/news/2016/10/7/better-supporting-a-growing-ros-industrial-software-platform)
```
Comment (gavanderhoorn) [inline README.md]: #48 suggests/asks whether this should be `consortium` instead.

Perhaps @schornakj can decide later today.

Comment (schornakj) [reply README.md]: @gavanderhoorn @ipa-rwu Based on further discussion of the support levels in IRC, let's use the `consortium` badge for Yak.

Comment (gavanderhoorn) [reply README.md]: @schornakj: you should be able to push new commits to this PR.

Comment (NadiaHG) [reply README.md]: I updated the badge, please give a final check and merge: https://github.com/ros-industrial/yak/pull/49/commits/494e2928d9bd65862fb514bcd4f62120d1f4fa5e

**Comment by schornakj** [comment]
@ipa-nhg Looks good! Thank you very much for the contribution.

**Pull request merged**